Sprint-3/todo-list/script.js

```diff
@@ -1,25 +1,126 @@
-function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
-}
-
-// These are the same todos that currently display in the HTML
-// You will want to remove the ones in the current HTML after you have created them using JavaScript
 let todos = [
   { task: "Wash the dishes", completed: false },
   { task: "Do the shopping", completed: false },
 ];
 
-populateTodoList(todos);
+let done = [];
+
+function populateTodoList() {
+  const todoList = document.getElementById("todo-list");
+  const completedList = document.getElementById("completed-todo-list");
+
+  // Clearing the existing list items
+  todoList.innerHTML = "";
+  completedList.innerHTML = "";
+
+  // Function to create list items
+  function createListItem(todo, index, isCompleted) {
+    const listItem = document.createElement("li");
+    listItem.className = 'todo-item';
```

Comment:
like I said in reading-list project, please define your constants at the beginning of your script not inside of your functions.